modules/core/src/main/scala/org/scalasteward/core/repocache/RepoCache.scala

```diff
@@ -26,7 +26,8 @@ import org.scalasteward.core.repoconfig.RepoConfig
 final case class RepoCache(
     sha1: Sha1,
     dependencyInfos: List[Scope[List[DependencyInfo]]],
-    maybeRepoConfig: Option[RepoConfig]
+    maybeRepoConfig: Option[RepoConfig],
+    maybeConfigParsingError: Option[String]
```

Comment:
I would name this field `maybeRepoConfigParsingError` to make it crystal clear that it is related to the above `maybeRepoConfig`.

The rest looks great to me. :+1:

Reply:
Good, thanks. I've renamed that field.